app/Console/Kernel.php

```diff
@@ -32,6 +32,6 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        //
+        $schedule->command('queue:work --sleep=3 --tries=3')->everyMinute();
```

Comment:
I think we should, James thinks we shouldn't.

In balance, we should probably keep it. I'm just warning that we may run into issues in the future.